fuzz/fuzz_targets/deserialize_br.rs

```diff
@@ -17,7 +18,31 @@ fuzz_target!(|data: &[u8]| {
 
     let mut allocator = Allocator::new();
     let program = node_from_bytes_backrefs(&mut allocator, &b1).unwrap();
-
+    let program_old = node_from_bytes_backrefs_old(&mut allocator, &b1).unwrap();
     let b2 = node_to_bytes_backrefs(&allocator, program).unwrap();
     assert_eq!(b1, b2);
+    let b3 = node_to_bytes_backrefs(&allocator, program_old).unwrap();
+    assert_eq!(b1, b3);
+});
+
+#[cfg(feature = "counters")]
+fuzz_target!(|data: &[u8]| {
+    let mut allocator = Allocator::new();
+    let program = match node_from_bytes_backrefs(&mut allocator, data) {
+        Err(_) => {
+            return;
+        }
+        Ok(r) => r,
+    };
+
+    let b1 = node_to_bytes_backrefs(&allocator, program).unwrap();
+
+    // reset allocators
+    let mut allocator = Allocator::new();
+    let mut allocator_old = Allocator::new();
```

Comment:
this fuzzer will be faster (and more efficient) if you use a single allocator that you reset (via `checkpoint()` and `restore_checkpoint()`). That way the memory allocations only need to be made once and can be reused.